tests/ui/cast.rs

```diff
@@ -47,4 +47,9 @@ fn main() {
     (-1i32).abs() as u32;
     (-1i64).abs() as u64;
     (-1isize).abs() as usize;
+    (-1i8).checked_abs() as u8;
+    (-1i16).checked_abs() as u16;
+    (-1i32).checked_abs() as u32;
+    (-1i64).checked_abs() as u64;
+    (-1isize).checked_abs() as usize;
```

Comment:
`checked_abs` returns `Option<T>` so we should use `unwrap` or something else to get contained values.

Reply:
Thank you for the review, i will fix that